Packs/CommonTypes/IndicatorFields/indicatorfield-Identity_class.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "indicator_identityclass",
+    "version": -1,
+    "modified": "2022-11-30T13:02:47.260146068Z",
+    "name": "Identity class",
+    "ownerOnly": false,
+    "description": "The type of entity that this Identity describes, e.g., an individual or organization.",
+    "cliName": "identityclass",
+    "type": "tagsSelect",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "selectValues": [
+        "individual",
+        "group",
+        "system",
+        "organization",
+        "class",
+        "unknown"
+    ],
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
```

Comment:
Should be associated to the Identity incident type?

Reply:
No, associated to all, we don't associate to specific types anymore.